### ISSUE TITLE
"Native" support for themes

### ISSUE THREAD

**Issue by bradberger** [alertifyjs/alertify.js#50]
Idea around this to add "native" support for various frameworks - Bootstrap, Purecss, Angular Material, etc. by customizing the templates to include proper classes for various elements in each framework.

This is in progress right now, but needs to be completed, and help is welcome. Right now a few frameworks I've added are below, but more are welcomed:

- Bootstrap
- Angular Material
- Pure CSS
- Material Design Light

**Comment by bradberger** [comment]
Need to update Bootstrap to include the default classes for the input element, too. See issue #52

**Comment by voidstate** [comment]
Can I suggest you use different Bootstrap classes?

Currently you have:

    this.dialogs.buttons.ok = "<button class='ok btn btn-success' tabindex='1'>{{ok}}</button>";
    this.dialogs.buttons.cancel = "<button class='cancel btn btn-danger' tabindex='2'>{{cancel}}</button>";

So cancel is btn-danger and ok is btn-success. To fit with the examples on the Bootstrap site, you should have btn-default for cancel and btn-primary for success.

**Comment by bradberger** [comment]
Agreed, that makes sense. If you want to submit a PR that would be great, otherwise I'll add it to the to-do list and get to it as soon as I can...

**Comment by igorsantos07** [comment]
+1 on this. I have a simple dialog and wanted to have a `btn-success` on it. Got around the issue uglily accessing the "private" object and changing the template directly, but would be way cooler if we could define custom classes - this could get in the way of the lib's lightness, tho.

Sidenote: there are no docs on this. Are you going to include the `.theme()` method on the docs page, or could I PR it? :) I just found it by accident, while fiddling with the source, trying to find a way to use bootstrap buttons.

**Comment by bradberger** [comment]
@igorsantos07 Go ahead and PR it, you're right it should be on the docs page. Same goes for the second parameter. That sounds like a good idea!

**Comment by brianespinosa** [comment]
@bradberger Is there a branch that this is being worked on from. I can assist with some of this as well. I would also like to suggest that [Semantic UI](https://github.com/Semantic-Org/Semantic-UI) be added to the list.

Additionally, though maybe not related to this issue, it would be great to be able to disable the injected CSS for alertify.js in another manner. There is no way to disable the injection when being used in React, unless I am missing something. I think it would make more sense to allow people to decide to include the CSS file, where they want to locate it, etc.

**Comment by bradberger** [comment]
@brianespinosa Thanks! The `master` branch is right now the current production version, that would be the one to fork and PR
